server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -85,6 +85,8 @@
   private long maxSeriesNumberAmongStorageGroup;
   private boolean initialized;
 
+  private Map<String, List<MeasurementSchema>> deviceTemplates = new HashMap<>();
```

Comment:
This should be persisted, you can refer to the persistence of mlog.txt
Recover this object when recovering